src/main/resources/templates/db_scripts/schema.json

```diff
@@ -19,29 +19,79 @@
   "tables": [
     {
       "tableName": "alert",
-      "generateId": true,
       "fromModuleVersion": 1.0,
       "withMetadata": true,
       "populateJsonWithId": true
     },
     {
       "tableName": "reporting_code",
-      "generateId": true,
       "fromModuleVersion": 1.0,
       "withMetadata": true,
       "populateJsonWithId": true
     },
     {
       "tableName": "purchase_order",
-      "generateId": true,
       "fromModuleVersion": 1.0,
       "withMetadata": true,
       "populateJsonWithId": true,
-      "customSnippetPath": "purchase_order_table.sql"
+      "customSnippetPath": "purchase_order_table.sql",
+      "index": [
+        {
+          "fieldName": "assignedTo",
+          "caseSensitive": false
```

Comment:
optional: The caseSensitive is already false by default, removing it makes it cleaner

Reply:
IMO making this explicit makes it clear w/o having any knowledge of the defaults, but I'm fine with it either way.